src/Lykke.Job.BitcoinPrivateWalletsCache.Services/NinjaFacade.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Common.Log;
+using Lykke.Job.BitcoinPrivateWalletsCache.Core.Domain;
+
+namespace Lykke.Job.BitcoinPrivateWalletsCache.Services
+{
+    public class NinjaFacade : INinjaFacade
+    {
+        private readonly ILog _log;
+        private readonly string _ninjaUrl;
+        
+        public NinjaFacade(ILog log, string ninjaUrl)
+        {
+            _log = log;
+            _ninjaUrl = ninjaUrl;
+        }
+
+        public event EventHandler<BalanceNotCalculatedEventArgs> BalanceNotCalculated;
+
+        public async Task SendBalanceRequest(string address, int attempt = 1)
+        {
+            try
+            {
+                await DoRequest($"{_ninjaUrl}balances/{address}/summary?colored=true");                
```

Comment:
The same about QBitNinja.Client, you can see example here
https://github.com/LykkeCity/bitcoinservice/blob/master/src/LkeServices/QBitNinja/QBitNinjaApiCaller.cs

Reply:
it's does not matter, it's temporary service.